tests/cypress/integration/components/a-z-nav.js

```diff
@@ -0,0 +1,46 @@
+describe('setting screen res', function () {
+
+  beforeEach(function () {
+    cy.viewport('macbook-15');
+
+  });
+
+  it('Navigating to A-Z Navigation', function () {
+
+    cy
+      .visit(Cypress.config().baseUrl + '/nav-a-z.html')
+
+  });
+
+
+  // checking for class name - nhsuk-c-a-z-nav__list
+  it('Checking for class name - nhsuk-c-a-z-nav__list', function () {
+    cy
+      .get('#nhsuk-nav-a-z').children().should('have.class', 'nhsuk-nav-a-z__list')
+
+  });
+
+  // checking for class name - nhsuk-c-a-z-nav__list-item-link
+  it('Checking for class name - nhsuk-nav-a-z__link', function () {
+    cy
```

Comment:
Can this go on the same line cy.visit()?

Reply:
updated it into single line